### PR TITLE
Feature multi org unit group persist indicators script

### LINKED ISSUE #40: Modify  "active/inactive OU" and "persist indicators" scripts to use multiple orgUnitGroups
Parse to orgunitgroups array the orgUnitGroup attribute using the token ";"

- [x]  active/inactive OU 
- [x] persist indicators

### PR DESCRIPTION
**Pull request by idelcano**
Closes #40
@adrianq I could not test persist-indicators script properly on my dev server. This [request](https://github.com/EyeSeeTea/organisationUnit-User-Assigment/blob/c48f85fe6d53a62fc809b7aecd2f3b96b5594b23/src/scripts/persist-indicators/index.js#L195) never ends. But I think it's independent of this PR.

### CODE REVIEW

File: src/scripts/persist-indicators/index.js [hunk 1]
```diff
@@ -80,19 +80,35 @@ AutoIndicatorsLoader.prototype.loadIndicator = function(indicator) {
...
-        //read data + post
-        _this.readAndPost(indicator);
+        if (indicator.orgUnitGroup != undefined) {
+            var orgUnitGroups = indicator.orgUnitGroup.split(";");
+            orgUnitGroups.forEach(function (orgUnitGroup) {
+                indicator.orgUnitGroup = orgUnitGroup;
+                _this.prepareAndPostIndicator(indicator);
+            });
+        } else {
+            this.prepareAndPostIndicator(indicator);
```
Comment (adrianq) [inline src/scripts/persist-indicators/index.js]: @idelcano shouldn't be _this? does it work like this?

**Comment by idelcano** [comment]
@adrianq you are right, your requested change solved
I tested it again, and i added other change to make work the query Params properly.

But i am not sure if it is correct. In my dev server the api query never obtains response.

Example of  api calls without any response or error: 

Multi orgUnitgroup:

http://localhost:8080/api/24/analytics.json?displayProperty=NAME&skipMeta=true&dimension=dx:OQgBQbtshRI&dimension=ou:OU_GROUP-BcVWNg98EGS&dimension=pe:201703;201702;201701;201612;201611;201610;201609;201608;201607;201606;201605;201604

http://localhost:8080/api/24/analytics.json?displayProperty=NAME&skipMeta=true&dimension=dx:OQgBQbtshRI&dimension=ou:OU_GROUP-HrO7HMn5SdV&dimension=pe:201703;201702;201701;201612;201611;201610;201609;201608;201607;201606;201605;201604

orgunit level and parent:

http://localhost:8080/api/24/analytics.json?displayProperty=NAME&skipMeta=true&dimension=dx:OQgBQbtshRI&dimension=ou:LEVEL-7;RZjIN6Adcdr&dimension=pe:201703;201702;201701;201612;201611;201610;201609;201608;201607;201606;201605;201604

**Review by adrianq — APPROVED** [review]

**Comment by josemp10** [comment]
Also tested and working! Many thanks guys!
@idelcano , I think that the problems you were having with the API calls were related to the number of org. units to be retrieved in the Myanmar database. 27.000 health facilities! 
I have done the testing for Myanmar and it also works, but it takes 5 hours....

**Comment by adrianq** [comment]
Thanks @josemp10 and @idelcano ! Merging this and releasing...

**Pull request merged**